.circleci/config.yml

```diff
@@ -22,6 +22,9 @@ jobs:
             mkdir -p tmp/
             curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./tmp/cc-test-reporter
             chmod +x ./tmp/cc-test-reporter
+            yarn install http-server -g
```

Comment:
I realize I gave you no guidance on how to modify CircleCI files! Sorry about that!

So we'll want to remove what you have modified and place the following before this [line](https://github.com/ifmeorg/ifme/blob/master/.circleci/config.yml#L130):

```
name: Deploy static Storybook site
command: |
  yarn storybook:circleci
```

We won't need to install http-server because placing the output in `/public/design` should render the site (through Rails magic).